src/tactic/doc_commands.lean

```diff
@@ -230,7 +237,12 @@ messages.
 
 -/
 @[user_command] meta def add_tactic_doc_command (_ : parse $ tk "add_tactic_doc") : parser unit :=
-parser.pexpr >>= of_tactic ∘ tactic.add_tactic_doc .
+do pe ← parser.pexpr,
+   elab ← of_tactic (elab_as_tde_or_error_msg pe),
```

Comment:
```suggestion
   elab ← of_tactic' (elab_as_tde_or_error_msg pe),
```
The `of_tactic` parser is broken and doesn't correctly propagate error messages.

Reply:
This file comes before `tactic.core` in the import hierarchy. `elab_as_tde_or_error_msg` is a workaround for this.